doc/source/ray-air/key-concepts.rst

```diff
@@ -47,6 +47,29 @@ Trainer objects will produce a :ref:`Results <air-results-ref>` object after cal
     :end-before: __air_trainer_output_end__
 
 
+Session
```

Comment:
Hmm we will want to move this once we have a basic tuner guide

Reply:
`Session` is relevant for both Tuner and Trainer. 
So I think it should have a space here. 
Individual component can still talk about Session separately in their own context.